botsettings.json

```diff
@@ -1,8 +1,8 @@
 {
-    "token" : "SERVER TOKEN",
+    "token" : "NzIyOTU3NTU5MzI5MzI1MTE2.Xuqosg.cXE9IwkWx8MGAuJ1k-boI_LkEbk",
```

Comment:
👏

Reply:
my guy just leaked his token?

Reply:
lmao

Reply:
I just saw this comment. pretty hight ahahaha 😁